frontend/src/modules/AppWrapper.tsx

```diff
@@ -19,60 +20,75 @@ import {
 } from "Src/pages";
 import { RouteProvider } from "./RouteProvider";
 
+function ErrorFallback({ error }) {
```

Comment:
can we put this under components

Reply:
Are u referring **_/src/components_** ? @palashgdev

Reply:
Yes

Reply:
ok @palashgdev

Reply:
Done!